tensorflow/python/ops/ctc_ops.py

```diff
@@ -789,17 +789,18 @@ def collapse_repeated(labels, seq_length, name=None):
   """Merge repeated labels into single labels.
 
   Args:
-    labels: Tensor of shape (batch, max value in seq_length)
-    seq_length: Tensor of shape (batch), sequence length of each batch element.
+    labels: Tensor of shape [batch, max value in seq_length]
+    seq_length: Tensor of shape [batch], sequence length of each batch element.
```

Comment:
??

Reply:
The shape of a tensor is denoted inside square brackets rather than round brackets for other parts of the documentation.